packages/ivy/ivy.1.3.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "gautier.hattenberger@enac.fr"
+version: "1.3.1"
+homepage: "http://www.eei.cena.fr/products/ivy"
+build: [
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "ivy"]
+  ["ocamlfind" "remove" "glibivy"]
+  ["ocamlfind" "remove" "tkivy"]
+]
+depends: "ocamlfind" {build}
+depexts: [
+  [["debian"] ["libpcre3-dev"]]
```

Comment:
Ok, will change that... just copied it from the older ivy version where someone else put it in like that.